Source/swiftlint/Extensions/TextTable+SwiftLint.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  TextTable+SwiftLint.swift
+//  SwiftLint
+//
+//  Created by Scott Hoyt on 2/4/16.
+//  Copyright © 2016 Realm. All rights reserved.
+//
+
+import SwiftLintFramework
+import SwiftyTextTable
+
+extension TextTable {
```

Comment:
since this is just used in `RulesCommand`, I think we can move it back in there. No need for this separate file.